front/components/sparkle/AppLayout.tsx

```diff
@@ -185,14 +214,23 @@ export default function AppLayout({
               </div>
               {titleChildren && SHOW_INCIDENT_BANNER && <IncidentBanner />}
             </div>
-
-            <div
-              className={classNames(
-                "flex h-full w-full flex-col",
-                isWideMode ? "items-center" : "max-w-4xl px-6"
-              )}
-            >
-              {loaded && children}
+            <div className="flex h-full w-full">
+              <div className="hidden items-center lg:visible lg:flex">
```

Comment:
Can't we have it living in the NavigationBar component like the hamburger menu for the mobile drawer?

Reply:
Seen IRL, we can use a `fixed  top-1/2 -translate-y-50` to display the button from the `NavigationBar` component.